src/login/logind-dbus.c

```diff
@@ -2382,6 +2383,80 @@ static int method_can_suspend_then_hibernate(sd_bus_message *message, void *user
                         error);
 }
 
+static int property_get_reboot_argument(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+        _cleanup_free_ char *parameter = NULL;
+        int r;
+
+        assert(bus);
+        assert(reply);
+        assert(userdata);
+
+        r = get_reboot_parameter(&parameter);
+        if (r != 0)
```

Comment:
we generally check with `< 0` for errors

Reply:
Ok. Changed as suggested.